app/views/admin/twitter_accounts/index.html.haml

```diff
@@ -2,7 +2,7 @@
   .col-sm-9.col-md-10
     %h1 Twitter Accounts
   .col-sm-3.col-md-2
-    = link_to "New Account", new_admin_twitter_account_path, class: "btn btn-success float-right"
+    = link_to "New Account", "/auth/twitter", class: "btn btn-success float-right"
```

Comment:
What makes this route?

Reply:
The `OmniAuth::Builder` block in  `config/initializers/omniauth.rb` -- they don't show up in the routes listing, but a `/auth/:provider` route gets defined for each provider.